source/usage-examples/bulkWrite.txt

```diff
@@ -7,14 +7,17 @@ Perform Bulk Operations
 You can perform bulk write operations on a collection by using the
 ``BulkWrite()`` method.
 
-The following example specifies a slice of ``WriteModels`` and an option
-to the ``BulkWrite()`` method, which performs the following in order:
-
-- Match a document where the ``title`` is "Record of a Shriveled Datum" and replace it with a new document
-- Match a document where the ``title`` is "Dodging Greys" and update that field to "Dodge The Greys"
+Example
+-------
 
 .. include:: /includes/usage-examples/run-example-tip.rst
 
+The following example performs the following in order on the ``haikus``
+collection:
+
+- Matches a document in which the ``title`` is "Record of a Shriveled Datum" and replace it with a new document
+- Matches a document in which the ``title`` is "Dodging Greys" and update that field to "Dodge The Greys"
```

Comment:
A couple items:
- "update" needs to have verb agreement
- it's not really the field, but rather a value attached to a field.

```suggestion
- Matches a document in which the ``title`` is "Dodging Greys" and updates that value to "Dodge The Greys"
```